src/data/markdown/translated-guides/en/02 Using k6/11 Environment variables.md

```diff
@@ -84,12 +84,22 @@ system environment variables with the same name.
 
 ## Environment variables k6 will read automatically
 
-k6 will also try to read some specific environment variables that the user can set to change the
-behavior of k6:
+k6 will also try to read some specific environment variables that the user can
+set to change the behavior of k6. These must be supplied to k6 at the system level
+(so not using `-e` or `--env`).
```

Comment:
can you link this to https://k6.io/docs/using-k6/options/#supply-environment-variables